lib/rate-limiter.js

```diff
@@ -12,15 +12,19 @@
  * @return {type}     returns a function that takes config options object
  */
 module.exports = function(db, app) {
+
   return function(opts) {
     var middleware = function(req, res, next) {
+      if (!db) {
```

Comment:
Actually reading more of the code `opts.ignoreErrors` may do this kind of thing.